.gitignore

```diff
@@ -5,3 +5,4 @@ __pycache__
 logs/
 chromium-bidi.iml
 node_modules/
+package-lock.json
```

Comment:
this file is created when we do some `npm` operations, so it's worth to explicitly exclude it

Reply:
We could also check in a `.npmrc` with `package-lock=false` in it to prevent npm from even touching the file.

Reply:
Is that your preference? I am happy to do either. We currently don't have a `.npmrc` file.

Reply:
The `.npmrc` feels strictly better since it avoids doing the unneeded busywork of reading/updating the `package-lock.json` altogether, but I guess the difference doesn't matter much in practice. We could even do both. No strong opinions here :)

Reply:
Thanks for the explanation. Done.